bin/steps/sqlite3

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set +e
+
+# shellcheck source=bin/utils
+source $BIN_DIR/utils
+
+SQLITE3_VERSION=$(apt-cache policy libsqlite3-dev | grep Candidate | sed 's/  Candidate: //')
+
+SQLITE3_VERSION_FILE="$BUILD_DIR/.heroku/python-sqlite3-version"
+if [ -f $SQLITE3_VERSION_FILE ]; then
+  INSTALLED_SQLITE3_VERSION=$(cat $SQLITE3_VERSION_FILE)
+fi
+
+# only install if the sqlite3 version has changed
+if [[ $INSTALLED_SQLITE3_VERSION != $SQLITE3_VERSION ]]; then
+  APT_CACHE_DIR="$CACHE_DIR/.heroku/python/apt/cache"
+  APT_STATE_DIR="$CACHE_DIR/.heroku/python/apt/state"
```

Comment:
I wonder if at some point in the future it would be worth adding some APT helper functions to buildpack-stdlib to reduce the duplication between the various buildpacks that have to play the non-system APT install dance?